glide-core/redis-rs/redis/src/cmd.rs

```diff
@@ -448,7 +448,7 @@ impl Cmd {
     ///
     /// This is useful for commands such as `SSCAN`, `SCAN` and others.
     ///
-    /// One speciality of this function is that it will check if the response
+    /// One specialty of this function is that it will check if the response
```

Comment:
The original is correct **English** spelling - your fix is the **American** spelling

Reply:
So what standard we follow? Not sure its important, i just follow the hints.